src/platforms/unity/native-support.mdx

```diff
@@ -16,30 +16,26 @@ Native support is `enabled` by default. The Unity SDK passes the options defined
 
 You can opt out of native support by unchecking `iOS Native Support` and `Android Native Support` in the configuration window.
 
-## Debug Symbols
+# Debug Symbols
```

Comment:
The title of the page is an H1 heading and each page can only have one H1 or it causes the right sidebar not to display properly. 

```suggestion
## Debug Symbols
```